src/structure/options.ts

```diff
@@ -22,6 +22,8 @@ export class StructureOptions extends OptionsGroup {
     public spaceFilling: HTMLOption<'boolean'>;
     /// should we show atoms labels
     public atomLabels: HTMLOption<'boolean'>;
+    /// should we show shapes
+    public shape: HTMLOption<'boolean'>;
```

Comment:
Why is this an option? I would assume we always want to show shapes if they are defined in the input

Reply:
Ok, fair enough